### PR TITLE
Upgrade samples to Akka 2.6.4 and Alpakka 2.0.0-RC1

### PR DESCRIPTION
**Pull request by ennru**

### CODE REVIEW

**Comment by ennru** [comment]
Travis looks tired.
I ran almost all examples and filed https://github.com/akka/alpakka/issues/2228

**Comment by ennru** [comment]
I pushed and update to wake Travis up.

Fixes #58

File: alpakka-sample-sqs-java/src/main/java/alpakka/sample/sqssample/Main.java [hunk 1]
```diff
@@ -54,13 +55,14 @@ public Main() {
 
     void run() throws Exception {
         // create SQS client
-        String sqsEndpoint = "this-uses-ElasticMQ";
+        String sqsEndpoint = "http://localhost:9324";
```
Comment (seglo) [inline alpakka-sample-sqs-java/src/main/java/alpakka/sample/sqssample/Main.java]: Was this change just for testing?

Comment (ennru) [reply alpakka-sample-sqs-java/src/main/java/alpakka/sample/sqssample/Main.java]: It is required to run the example, but wasn't before?!

Comment (seglo) [reply alpakka-sample-sqs-java/src/main/java/alpakka/sample/sqssample/Main.java]: I guess?  Working code has my vote :+1:

**Pull request merged**